src/NServiceBus.Core/Unicast/UnicastBusConfig.cs

```diff
@@ -52,6 +52,10 @@ public string DistributorDataAddress
         /// Gets/sets the address to which messages received will be forwarded.
         /// </summary>
         [ConfigurationProperty("ForwardReceivedMessagesTo", IsRequired = false)]
+        [ObsoleteEx(
+            TreatAsErrorFromVersion = "6",
+            RemoveInVersion = "7",
+            ReplacementTypeOrMember = "EndpointConfiguration.ForwardReceivedMessagesTo")]
```

Comment:
To get they would have to use `context.Settings.Get<string>("forwardReceivedMessagesTo")` so lets go with "Use xyz to configure the address and abc to get the current value"?